### DISCUSSION TITLE
Ability to add a web page / article to read later

### DISCUSSION THREAD

**Discussion by mptpro** [spacecowboy/Feeder#305] [Ideas]
If this function already exisits, I apologize for the post - I searched but didn't find this.

I'm looking for a way to save a web page to read later in Feeder.

**Comment by spacecowboy** [comment]
Open the menu and click "Save article" with the Star icon

**Reply by mptpro** [reply]
What is I mean is from a web page in a browser, not from within the Feeder app.  Something like sending an article to the Feeder app to view it later.

**Reply by spacecowboy** [reply]
No. That's not what Feeder is. You can't add random web sites to it.

**Reply by spacecowboy** [reply]
Use something like Mozilla's Pocket for that
